protos/dcs/tts/v0/tts.proto

```diff
@@ -0,0 +1,94 @@
+syntax = "proto3";
+package dcs.tts.v0;
+import "dcs/common/v0/common.proto";
+option csharp_namespace = "RurouniJones.Dcs.Grpc.V0.Tts";
+option go_package = "github.com/DCS-gRPC/go-bindings/dcs/v0/tts";
+
+service TtsService {
+  rpc Transmit(TransmitRequest) returns (TransmitResponse) {}
+}
+
+message TransmitRequest {
```

Comment:
While the PR isn't ready for review yet, @rurounijones feel free to give the proto definition a look already (we discussed a few things in voice a while ago, but I unfortunately don't remember everything we agreed on)